Dockerfile.sdk

```diff
@@ -59,7 +59,8 @@ RUN apt-get update && \
             python3-dev \
             rapidjson-dev \
             vim \
-            wget && \
+            wget 
+            wkhtmltopdf && \
```

Comment:
Shouldn't this be `python3-pdfkit`?
Additionally the same can be removed from **L0_kibana_summary/test.sh**

Reply:
I see I will change this

Reply:
I think we install pdfkit as the dependency in `requiremenets.txt`. If `wkhtmltopdf ` is not required, we can probably remove it from the MA docker file too.

Reply:
For the kibana summary I did not need to manually install wkhtmltopdf. It was installed with python-pdfkit as a dep. as I recall.

Reply:
@aramesh7 can you create a gitlab PR to remove this install from the kibana summary's test.sh ?

Reply:
Done

Reply:
I do not see an MR for this on Gitlab